docs/pages/updates/index.tsx

```diff
@@ -715,7 +715,7 @@ export default function WhatsNew() {
             <li>
               Exchanged <InlineCode>deploy</InlineCode>, <InlineCode>reset</InlineCode> and{' '}
               <InlineCode>generate</InlineCode> commands for{' '}
-              <InlineCode>keystone-next prisma</InlineCode> commands.
+              <InlineCode>keystone prisma</InlineCode> commands.
```

Comment:
I would think we wouldn't want to update this(and the other bit in this file) given it's about a release and in that release, the command was `keystone-next`?

Reply:
ah good catch, missed this exclude in the find-replace